### ISSUE TITLE
While creating multiple volumes, some commands are called without name parameter value

### ISSUE THREAD

**Issue by simkam** [RedHatQE/teflo_openstack_client_plugin#13]
While creating multiple volumes, some commands are called without name value:
```
provision:
  - name: ceph-node-volume
    provisioner: openstack-client
    credential: openstack-creds
    volume:
      max: 4
      size: 20
```

```
2022-04-07 14:26:08,833 INFO [teflo_openstack_client_plugin.os_client_provisioner.create:345] Provisioning Asset ceph-node-volume.
2022-04-07 14:26:08,837 INFO [ospclientsdk.load_cloud_config:71] Successfully loaded cloud credentials.
2022-04-07 14:26:08,838 DEBUG [ospclientsdk.func:134] {'size': 20, 'res': 'ceph-node-volume-1'}
2022-04-07 14:26:08,838 DEBUG [ospclientsdk.helpers.decorators.run_cmd_raw:100] command: openstack volume create --size 20 ceph-node-volume-1 -f json
2022-04-07 14:26:08,838 DEBUG [ospclientsdk.func:134] {'size': 20, 'res': 'ceph-node-volume-2'}
2022-04-07 14:26:08,839 DEBUG [ospclientsdk.func:134] {'size': 20, 'res': 'ceph-node-volume-2'}
2022-04-07 14:26:08,839 DEBUG [ospclientsdk.helpers.decorators.run_cmd_raw:100] command: openstack volume create --size 20 ceph-node-volume-2 -f json
2022-04-07 14:26:08,839 DEBUG [ospclientsdk.func:134] {'size': 20}
2022-04-07 14:26:08,839 DEBUG [ospclientsdk.helpers.decorators.run_cmd_raw:100] command: openstack volume create --size 20 ceph-node-volume-4 -f json
2022-04-07 14:26:08,842 DEBUG [ospclientsdk.helpers.decorators.run_cmd_raw:100] command: openstack volume create --size 20  -f json
2022-04-07 14:26:10,333 ERROR [teflo_openstack_client_plugin.os_client_provisioner._run_action:329] Command did not execute successfully
2022-04-07 14:26:11,808 ERROR [teflo.provisioners.asset_provisioner.create:150] usage: openstack volume create [-h] [-f {json,shell,table,value,yaml}]
                               [-c COLUMN] [--noindent] [--prefix PREFIX]
                               [--max-width <integer>] [--fit-width]
                               [--print-empty] [--size <size>]
                               [--type <volume-type>]
                               [--image <image> | --snapshot <snapshot> | --source <volume>]
                               [--description <description>]
                               [--availability-zone <availability-zone>]
                               [--consistency-group consistency-group>]
                               [--property <key=value>] [--hint <key=value>]
                               [--bootable | --non-bootable]
                               [--read-only | --read-write]
                               <name>
openstack volume create: error: the following arguments are required: <name>
```

```
$ teflo --version
teflo, version 2.2.3

$ pip show python-openstackclient
Name: python-openstackclient
Version: 5.8.0
```

**Comment by dno-github** [comment]
Jira ticket CCITCARBON-434 has been created for this issue. Url : https://issues.redhat.com/browse/CCITCARBON-434

**Comment by JunqiZhang0** [comment]
solved by #18
